android/src/main/java/com/internxt/mobilesdk/services/FileSystem.kt

```diff
@@ -0,0 +1,76 @@
+package com.internxt.mobilesdk.services
+import android.net.Uri
+import com.internxt.mobilesdk.utils.FileAccessRejectionException
+import java.io.BufferedReader
+import java.io.File
+import java.io.FileReader
+import java.io.IOException
+
+// This class is called FS to avoid naming conflicts with Java FileSystem class
+object FS {
+
+  /**
+   * Check if a file exists at a given path
+   */
+  fun fileExists(path: String): Boolean {
+    return File(path).exists()
+  }
+
+
+  /**
+   * Unlink a file only if it exists
+   *
+   * Return true if the file is successfully deleted, false otherwise
+   */
+  fun unlinkIfExists(path: String): Boolean {
+    val file = File(path)
+    val exists = this.fileExists(path)
+
+    try {
+      if(exists) {
+        return file.delete()
+      }
+
+      return false
```

Comment:
If you return false at the end, this returns here and on the catch are not needed then